starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,63 @@
 // Names and Input
+let hacker1 = "Charpentier"
+console.log("The driver's name is " + hacker1);
 
+let hacker2 = prompt("What's your name?");
+console.log("The navigator's name is " + hacker2);
 
 //Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log("The Driver has the longest name, it has " + hacker1.length + " characters");
+} else if (hacker1.length === hacker2.length) {
+    console.log("wow, you both got equally long names, " + hacker1.length + " characters!!");
+} else {
+    console.log("Yo, navigator got the longest name, it has " + hacker2.length + " characters");
+}
 
+// Print name
+let lettres = ""
+
+for(i = 0; i < hacker1.length; i++) {
+    console.log(lettres += hacker1.charAt(i).toUpperCase() + " ")
```

Comment:
Care, here you are doing a console log inside a loop, so you will print something at every round of the loop. It is a good idea to start from letter which is empty and add letters to it ! The console log should be right after ;)